src/components/Plugin/ContributionRewardExtRewarders/rewardersProps.ts

```diff
@@ -61,3 +61,21 @@ export const getCrxRewarderComponent = (pluginState: IContributionRewardExtState
     return null;
   }
 };
+
+/**
+ * defined but not exported by arc.js
+ */
+export interface IEntityState {
+  id: string;
+}
+
+export const getCrxRewarderProposalClass =
+  (pluginState: IContributionRewardExtState): new (context: Arc, idOrOpts: string | IEntityState) => any => {
+
+    const contractName = rewarderContractName(pluginState, false);
+
+    switch (contractName) {
```

Comment:
can this be `switch (rewarderContractName(pluginState, false)) {`

Reply:
Done